va/validation-authority.go

```diff
@@ -659,21 +669,27 @@ func (va *ValidationAuthorityImpl) getCAASet(hostname string) (*CAASet, error) {
 		if _, present := policy.PublicSuffixList[label]; present {
 			break
 		}
-		CAAs, _, err := va.DNSResolver.LookupCAA(label)
+		CAAs, caaRtt, err := va.DNSResolver.LookupCAA(label)
```

Comment:
Oh, this is supposed to be part of this diff?

This makes it more clear that the RPCMonitor is not the right abstraction for this change.